R/location.R

```diff
@@ -203,18 +203,31 @@ orderly_location_remove <- function(name, root = NULL, locate = TRUE) {
 ##'
 ##' @inheritParams orderly_metadata
 ##'
-##' @return A character vector of location names. The special name
-##'   `local` will always be present.
+##' @param verbose Logical, indicating if we should return a
+##'   data.frame that includes more information about the location.
+##'
+##' @return Depending on the value of `verbose`:
+##'
+##' * `verbose = FALSE`: A character vector of location names. The
+##'   special name `local` will always be present.  This is the
```

Comment:
This line about local should be moved out of the bullet points into the general description.

As it stands it seems to imply that with `verbose=TRUE` the return value does not include local.